src/app/features/course/new-course.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'new-course',
+  templateUrl: './new-course.component.html',
+  styleUrls: ['./course.component.css', '../../shared/components/shared.style.css']
+})
+export class NewCourseComponent implements OnInit {
+
+  btnText = 'Create course';
+  btnAuthorText = 'Create author';
+  btnDelAuthorText = 'Delete author';
+  hasError = false;
+  hasError1 = false;
+  hasError2 = false;
+  hasError3 = false;
+
+  form: FormGroup;
+
+  constructor() {
+    this.form = new FormGroup({
+      courseTitle: new FormControl(null),
```

Comment:
For reactive forms better to add validation here in form control e.g 
`new FormControl('', [Validators.required()])`

Reply:
also form initialising can be moved to ngOnInit() method